backup_and_restore/application_backup_and_restore/installing/about-oadp-data-mover.adoc

```diff
@@ -1,27 +1,27 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="about-oadp-1-3-data-mover"]
-= About the OADP 1.3 Data Mover
+[id="about-oadp-data-mover"]
+= About the {oadp-short} Data Mover
 include::_attributes/common-attributes.adoc[]
-:context: about-oadp-1-3-data-mover
+:context: about-oadp-data-mover
 
 toc::[]
 
-OADP 1.3 includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc#oadp-about-kopia[Kopia] as the uploader mechanism to read the snapshot data and write to the unified repository.
+{oadp-short} includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc#oadp-about-kopia[Kopia] as the uploader mechanism to read the snapshot data and write to the unified repository.
 
-OADP supports CSI snapshots on the following:
+{oadp-short} supports CSI snapshots on the following:
 
-* Red Hat OpenShift Data Foundation
+* {odf-full}
 * Any other cloud storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
 
 [IMPORTANT]
 ====
-The OADP built-in Data Mover, which was introduced in OADP 1.3 as a Technology Preview, is now fully supported for both containerized and virtual machine workloads.
+The {oadp-short} built-in Data Mover, which was introduced in {oadp-short} 1.3 as a Technology Preview, is now fully supported for both containerized and virtual machine workloads.
 ====
 
-[id="enabling-oadp-1-3-data-mover"]
+[id="enabling-oadp-data-mover"]
```

Comment:
I know you're only touching this ID to remove the version number, so this is out of scope, but - if you want to do some clean up in this PR or another, technically all IDs (that are not the top level ID of an assembly) need a context at the end:
```suggestion
[id="enabling-oadp-data-mover_{context}"]
```
Not necessary to do in this case since this is already an existing ID, but just something to keep in mind for future doc efforts